src/hidimstat/utils.py

```diff
@@ -24,13 +24,16 @@ def quantile_aggregation(pvals, gamma=0.5, gamma_min=0.05, adaptive=False):
 
 def fdr_threshold(pvals, fdr=0.1, method="bhq", reshaping_function=None):
```

Comment:
While we're at it, the docstring seems to be missing...

Reply:
I add it.